jsonschema-csr/src/vocabularies/applicator/all_of.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{compilation::edges::EdgeLabel, vocabularies::Keyword, JsonSchema};
+use serde_json::Value;
+use std::ops::Range;
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct AllOf {
+    keywords: Range<usize>,
+}
+
+impl AllOf {
+    pub(crate) fn build(start: usize, end: usize) -> Keyword {
+        Self {
+            keywords: start..end,
+        }
+        .into()
+    }
+}
+
+impl AllOf {
+    pub(crate) fn is_valid(&self, schema: &JsonSchema, instance: &Value) -> bool {
+        schema.keywords[self.keywords.clone()]
```

Comment:
It should go over edges as each item inside `allOf` is a schema, which is a range of keywords in this layout.